retrofit-mock/src/main/java/retrofit2/mock/Calls.java

```diff
@@ -42,7 +42,7 @@
     return new FakeCall<>(response, null);
   }
 
-  public static <T> Call<T> failure(IOException failure) {
+  public static <T> Call<T> failure(Exception failure) {
```

Comment:
This is a backwards incompatible change and needs to be introduced as an overload. Also it probably should take `Throwable`.

Reply:
Probably nicer as an overload (did this as part of a98b705) but this is still backwards compatible as written right? Tests didn't fail on us.